gobblin-cluster/src/test/java/org/apache/gobblin/cluster/GobblinHelixJobSchedulerTest.java

```diff
@@ -129,58 +148,176 @@ public void setUp()
     this.thread.start();
   }
 
+  // Time span exceeds throttle timeout, within same workflow, throttle is enabled
+  // Job will be updated
```

Comment:
Comments describing the method should be a java doc instead of a regular comment